src/main/scala/viper/server/core/VerificationWorker.scala

```diff
@@ -175,6 +175,7 @@ class ViperBackend(val backendName: String, private val _frontend: SilFrontend,
     if (!_frontend.prepare(argsWithDummyFilename)) {
       return
     }
+    _frontend.resetPlugins()
```

Comment:
why is this needed / was not needed before?

Reply:
This is needed to a) select which plugins are active based on the configuration that was just created from the passed arguments (previously I think ViperServer essentially always used the default ones only here), and b) to forward the configuration object to the plugins so they can read them (previously they got a null-config, which caused #184). I added a comment for this as well.